web/testing/commonHeader.js

```diff
@@ -53,46 +53,50 @@
         prefix = prefix + base_prefix;
       }
     }
-    
+
     // The first keyboard added will be the default keyboard for touch devices.
     // For faster loading, it may be best for the default keyboard to be
     // locally sourced.
     kmw.addKeyboards({id:'us',name:'English',languages:{id:'en',name:'English'},
       filename:(prefix + 'us-1.0.js')});
-      
+
     // Add more keyboards to the language menu, by keyboard name,
     // keyboard name and language code, or just the BCP-47 language code.
     // We use a different loading pattern here than in the samples version to provide a slightly different set of test cases.
     kmw.addKeyboards('french','@he');
     kmw.addKeyboards({id:'sil_euro_latin', name:'SIL EuroLatin', languages: [{id:'no'}, {id:'sv'}]}); // Loads from partial stub instead of the compact string.
-  
+
     // Add a keyboard by language name.  Note that the name must be spelled
     // correctly, or the keyboard will not be found.  (Using BCP-47 codes is
     // usually easier.)
-    kmw.addKeyboardsForLanguage('Dzongkha');
-    
-    // Add a fully-specified, locally-sourced, keyboard with custom font  
+    kmw.addKeyboardsForLanguage('Dzongkha').then(result => {
+      console.log('Adding Dzonkha: ', result);
+    }).catch(error => {
+      alert(error);
+    });
```

Comment:
I'd rather have this either on all or on none of the add-keyboard calls, not just on some.  This is a _common_ header used by many of the testing pages, so it should be standardized and consistent with how it loads any keyboard.

It may be 'simpler' to leave this out on this page and create a custom-tailored test page for custom `.catch()` blocks.

Reply:
Though... I suppose `addKeyboards` doesn't yet have `Promise` implementation, does it?  Just `addLanguageKeyboards`.  Huh, chose the hard one first, eh?

Reply:
I've split this to a separate test page "Promise API"